proto/vtctlservice.proto

```diff
@@ -28,3 +28,13 @@ import "vtctldata.proto";
 service Vtctl {
   rpc ExecuteVtctlCommand (vtctldata.ExecuteVtctlCommandRequest) returns (stream vtctldata.ExecuteVtctlCommandResponse) {};
 }
+
+// Service Vtctld exposes gRPC endpoints for each vt command.
+service Vtctld {
+  // GetKeyspace reads the given keyspace from the topo and returns it.
+  rpc GetKeyspace(vtctldata.GetKeyspaceRequest) returns (vtctldata.Keyspace) {};
+  // GetKeyspaces returns the names of all keyspaces in the topo.
+  rpc GetKeyspaces(vtctldata.GetKeyspacesRequest) returns (vtctldata.GetKeyspacesResponse) {};
```

Comment:
Is GetKeyspaces not returning the actual object because that's not how the existing rpc works? I would vote for this call to return the list of keyspace objects that you've currently created a new RPC for. If we want to be able to restrict the data that is returned, I would prefer to use a view or a fieldmask. https://google.aip.dev/157

Reply:
Yeah that was originally my reason. What do you think about removing the "just the names" rpc entirely, and having clients do `for _, ks := range kss { ks.Name }` if they want?

I think the downside to using a fieldmask in this specific case is the "subset" of fields we want is actually a list of scalars, so our return types should either be `[]string` or `[]*pb.Keyspace`, which I'm not sure how to represent nicely in protos.

Reply:
Agreed, my vote is to return the whole object as `[]*pb.Keyspace` and eliminate the `ShowAllKeyspaces` rpc

Reply:
Semi-related, it's weird that the underlying `topodata.Keyspace` proto doesn't have the name easily accessible, making you wrap that message here.

Reply:
Yeah I agree. I wonder how controversial of a change it would be to put a `name` field in the `Keyspace` and `Shard` protos or if that would require a more careful migration. If the latter I'd rather stick with this thin wrapper for now and revisit later.

@deepthi do you have any thoughts here?

Reply:
To the first point, all rpcs take a `Request` proto and return a `Response` proto. This allows us to change the contents of the proto between versions without breaking compatibility. 
Regarding adding `name` to the proto objects: AFAICT the logic seems to have been that the `name` of the object is the filename and the object is a representation of the file contents. This makes perfect sense from the POV of the topo - why duplicate information and why not keep the data small.
As we expand the usage of these proto objects beyond topo calls and start passing them around, it will be nice to not have to wrap them with another object that has the name, so I'm not opposed to this change. We can rely on protobuf compatibility guarantees.
@sougou @enisoc please chime in if you disagree.